Sources/SwiftProtobuf/TextFormatScanner.swift

```diff
@@ -216,6 +216,25 @@ internal struct TextFormatScanner {
         }
     }
 
+    private mutating func parseUTF8Identifier() -> UnsafeBufferPointer<UInt8>? {
```

Comment:
Nice catch!

While changing this, I noticed that `parseIdentifier()` just below should also be non-optional.  While fixing that, I realized that this exact loop was being repeated in several places, so I edited them to use `parseUTF8Identifier()`, simplifying a number of points in the code.

Thanks!